CHANGELOG.asciidoc

```diff
@@ -45,6 +45,9 @@ See Conventional Commits (https://conventionalcommits.org) for commit guidelines
 [float]
 ===== Features
 * Events are now beaconed when the user abandons or backgrounds the page, this makes sure the agent never miss an event. Before this change it was only sending them every interval configurable via `flushInterval`. Unsupported browsers: Firefox, IE11: {pull}1146[#1146]
+
+NOTE: Transactions based on the <<supported-technologies,supported technologies>> and <<custom-managed-transactions, custom managed transactions>> are the only ones taken into account.
+
```

Comment:
Here I have chosen to use the "NOTE" style to describe the nuance. It is something that I have seen in our documentation and I consider it a good way to highlight things such as exceptions, and nuances of something

The screenshot below shows us how it will look like once the PR is merged:

<img width="851" alt="Screenshot 2022-05-25 at 12 12 43" src="https://user-images.githubusercontent.com/15065076/170253408-39c90c6e-ddd4-471c-bf69-7fa18d6d8fef.png">

What do you think? @vigneshshanmugam @kyungeunni

Reply:
So cool, I love it.